README.md

```diff
@@ -1,42 +1,80 @@
 
-  - Open an issue on this repository describing your proposal.
-  - Gather support from other contributors, including ideally at least one or two browser vendors.
-  - Proposals with enough support and a clear technical direction can request that a repository be created under the immersive-web organization. (Alternatively an external repository can be transferred to the immersive-web organization.) The repo will be labeled as being in incubation.
-  - Development of the idea will continue on the repository until it's ready to be integrated into another incubation or promoted out of incubation state (e.g., to a WG).
-  
-## What belongs here vs. in another repo/standards group/issue tracker
-There are a lot of different places that ideas for web standards could be proposed, so at times it may not be clear where the best place is to start. There are no hard and fast rules, but here are some things to consider:
+## My proposal Issue has widespread support and we've found consensus. Now what?
 
-  - If your proposal isn't in some way related to Immersive Web technologies (Virtual Reality, Augmented Reality, Mixed Reality, etc.), then this probably isn't the place for it. Consider checking out the [W3C Community and Business Groups](https://www.w3.org/community/) if you don't know of an existing incubation or specification that your proposal fits under. The [Web Incubator Community Group (WICG)](https://wicg.io/) is a good general-purpose venue.
-  - Similarly, if your proposal is related to the Immersive Web but useful in other contexts as well (i.e., Geolocation improvements), then it should be probably be handled by a group that's less exclusively focused on Immersive Web tech. Feel free to add a note where appropriate to let the Immersive Web community know about a relevant effort.
-  - If you have an issue with a specific spec, open an issue on that spec first.
+Terrific! You did the hard work of getting enough momentum in the group to figure out what you're making and how it would work. The next step is to recruit one or two people from other organizations who want to join you as feature advocates. This isn't a formal W3C community group role; it simply means that each of you agrees to do the work of nailing down a feature's details and working with the rest of the group to find consensus on its details.
 
-So what happens if you get it wrong? Nothing bad! We'll point you towards a more appropriate venue if necessary and, when possible, help migrate the issue for you. And please don't assume that a request to continue discussion of your issue elsewhere is a criticism, we genuinely want to ensure that your idea is reaching the right audience!
+Once two or three people from separate organizations have volunteered to be feature advocates then you should reach out to the CG chair and ask that a feature repo be created.
```

Comment:
Is "you need two or three people from separate orgs signed up to be feature advocates first" really going to be the bar for creating a repo?  Because you'll end up pushing a lot of people into personal repos for feature design up front.

Maybe I'm just reacting to the difference between "you need at least one or two other people saying this is interesting to discuss" (which is the WICG bar) and "two or three others have volunteered to be feature advocates" which sounds like they're 100% bought in to the feature (and feels like a high bar).